lightning/src/ln/offers_tests.rs

```diff
@@ -1724,3 +1725,85 @@ fn fails_paying_invoice_more_than_once() {
 	let invoice_error = extract_invoice_error(alice, &onion_message);
 	assert_eq!(invoice_error, InvoiceError::from_string("DuplicateInvoice".to_string()));
 }
+
+/// This test verifies that both the blinded message paths and blinded payment
+/// paths are properly padded and function as expected.
+#[test]
+fn test_blinded_path_padding() {
```

Comment:
Could we instead write unit tests in the `blinded_path` module? Shouldn't need this much setup to test the new behavior.

Reply:
Thanks a lot for the pointer, Jeff!

I experimented with writing tests in the `blinded_path` module, but seems like it required a bit of setup as well.
So for now I have moved the testing to `onion_message/functional_test.rs` which helped make the setup much simpler!

Let me know if the test looks good! If we still prefer to keep testing the padding with the `blinded_path` module, just let me know-I would love to make the change!

Thanks a lot!

Reply:
Yep, that works.